### DISCUSSION TITLE
Oracle Schema Build Issue

### DISCUSSION THREAD

**Discussion by caox2** [TPC-Council/HammerDB#130] [Q&A] [answered]
I couldn't find any information regarding this error:

Error in Virtual User 1: wrong # args: should be "do_tpcc system_user system_password instance count_ware tpcc_user tpcc_pass tpcc_def_tab tpcc_ol_tab tpcc_def_temp partition timesten hash_clusters num_vu"

Using version 3.3

**Comment by sm-shaw** [comment]
In the build script the very last line is what is used to call "do_tpcc". Please copy this last line. Most likely there is a space in one of the parameters such as a password so this is being interpreted as an extra argument.  If you want to include special characters wrap the argument in curly brackets eg {}

**Comment by caox2** [comment]
Hi Steve,

This is the line from the script.  I'm guessing its space between the servicename and the 1?

"do_tpcc sys as sysdba syspassword oracle  1 tpcc tpcc tpcctab tpcctab temp false false false 1"

**Comment by sm-shaw** [comment, marked as answer]
OK so the problem is the username is given as "sys as sysdba" so it is trying to build a connect string of  "sys as sysdba/syspasswd@oracle" which is incorrect. Please use the system user not sys so the connect string is "system/systempasswd@oracle" it shows the error too many arguments as it detects sys, as and sysdba all as separate arguments.

**Comment by caox2** [comment]
Ah that is it!  Thanks for the help Steve!
